lib/checks/050-koenig-css-classes.js

```diff
@@ -20,8 +20,12 @@ checkKoenigCssClasses = function checkKoenigCssClasses(theme, options, themePath
 
     _.each(ruleSet, function (check, ruleCode) {
         if (theme.files) {
-            // Cheeck CSS files for presence of the classes
+            // Check CSS files and HBS files for presence of the classes
             _.each(theme.files, function (themeFile) {
+                if (['.css', '.hbs'].indexOf(themeFile.ext) === -1) {
```

Comment:
would `includes` be more readable here? :shrimp:

Reply:
had exact same comment yesterday :smile:  only noticed today that you have to 'submit review' manually on Github for the comment to go out :confused: